src/io/OhmmsData/OhmmsParameter.h

```diff
@@ -46,10 +46,11 @@
 
 enum class TagStatus
 {
-  OPTIONAL, // there is a default value
-  REQUIRED, // no default value, input required.
+  OPTIONAL,   // there is a default value
+  REQUIRED,   // no default value, input required.
   DEPRECATED, // deprecated input tag. Input becomes optional
-  DELETED // deleted input tag. Hard stop if detected.
+  DELETED,    // deleted input tag. Hard stop if detected.
+  NOSUPPORT   // not supported input tags due to incompatible changes. Hard stop if detected.
```

Comment:
Prefer UNSUPPORTED here, it is a little clearer imo.

Reply:
Changed.